### ISSUE TITLE
Bug in `griddap()` 

### ISSUE THREAD

**Issue by sckott** [ropensci/rerddap#71]
```r
library(rerddap)
x=griddap("NOAA_DHW_5km",
           time=c("2017-12-30T00:00:00Z", "2017-12-31T00:00:00Z"),
           latitude=c(18,19),
           longitude=c(-161,160),
           url = "http://oos.soest.hawaii.edu/erddap/",
           fmt = "csv",
           read = TRUE)
# Error: HTTP Status 500 - Query error: For variable=CRW_DHW axis#1=latitude Constraint="[(18):1:(19)]": StartIndex=1440 is greater than StopIndex=1420.

y=griddap(aa,
            time=c("2017-12-30T00:00:00Z", "2017-12-31T00:00:00Z"),
            latitude=c(19,18),
            longitude=c(-161,160),
            url = "http://oos.soest.hawaii.edu/erddap/",
            fmt = "csv",
            read = TRUE)
# Error: HTTP Status 500 - Query error: For variable=CRW_DHW axis#1=latitude Constraint="[(18):1:(19)]": StartIndex=1440 is greater than StopIndex=1420.
```

**Comment by sckott** [comment]
the `/info` ERDDAP request is returning the wrong information. 

e.g. https://upwell.pfeg.noaa.gov/erddap/info/noaa_esrl_d7ee_ffb3_d5a0/index.json 

is returning for latitude 

```
["attribute", "latitude", "actual_range", "float", "-89.75, 89.75"],
```

which is in the opposite order from what the web interface has https://upwell.pfeg.noaa.gov/erddap/griddap/noaa_esrl_d7ee_ffb3_d5a0.html 

<img width="708" alt="screen shot 2018-03-29 at 2 13 33 pm" src="https://user-images.githubusercontent.com/577668/38113940-649cabbc-335b-11e8-9dab-3e12c4017f5f.png">


we definitely depend on the order of the min and max of lat and long to be in the same order that the ERDDAP server expects, so AFAICT we're at a dead end here unless maybe the ERDDAP `/info` request is returning the wrong data, and it can be fixed?

thoughts @rmendels ?

**Comment by sckott** [comment]
should be fixed in #68
